docs/security.html

```diff
@@ -609,7 +609,7 @@ <h3><a id="security_sasl" href="#security_sasl">7.3 Authentication using SASL</a
 
             </li>
             <tt>KafkaServer</tt> section in the JAAS file tells the broker which principal to use and the location of the keytab where this principal is stored. It
-            allows the broker to login using the keytab specified in this section. See <a href="#security_sasl_brokernotes">notes</a> for more details on Zookeeper SASL configuration.
+            allows the broker to login using the keytab specified in this section. See <a href="#security_jaas_broker">notes</a> for more details on Zookeeper SASL configuration.
```

Comment:
Fix the broken link. The old link: `#security_sasl_brokernotes` was pointing to the new link: `#security_jaas_broker` based on the old commit: https://github.com/apache/kafka/pull/1232/files#diff-64f03d45aabc1a9f07a427f4f21d28f5R215 
VS. 
our current file: https://github.com/apache/kafka/blob/trunk/docs/security.html#L425